nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddPermission.java

```diff
@@ -58,6 +59,15 @@
 
     public static final String COMMENT_KEY = "comment";
 
+    /**
+     * Define whether virtual user (non-existent user) is allowed in Nuxeo automation. If allowed, Nuxeo server will
+     * not check the user existence during automation execution. Set this property to true if you use Nuxeo computed
+     * user or computed group.
+     *
+     * @see <a href="https://jira.nuxeo.com/browse/NXP-21889">NXP-21889</a>
```

Comment:
The Javadoc should mention that this is a _Configuration Property_ name.
Don't link to JIRA in Javadoc.
Add a `@since 9.1`.